### ISSUE TITLE
Expose .centerAnchors property

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/Anchorage#8]
On the pattern of `horizontalAnchors` and `verticalAnchors`:

```swift
view1.centerAnchors == view2.centerAnchors
// same as:
view1.centerXAnchor == view2.centerXAnchor
view1.centerYAnchor == view2.centerYAnchor
```

**Comment by ZevEisenberg** [comment]
I just came here to file this bug, but found that I already had. Thinking I should probably make it happen.

**Comment by KingOfBrian** [comment]
Fixed in 3.0!
